### ISSUE TITLE
csharp control library

### ISSUE THREAD

**Issue by OpenSourceCS** [hadouken/hadouken#194]
a csharp control library to control hadouken from another program would be pretty useful. Btw this is the most aspiring headless torrent client project i have ever seen in my life. Good work guys.

**Comment by bitterypaul** [comment]
I support this request. This feature can be useful for manipulating hadouken from another application. This can be implemented as a separate project. The work is simple. Just creating a wrapper for the jsonrpc api of hadouken.

**Comment by vigneshkarthisan** [comment]
I think this is necessary to create a project for this. I like and support this idea. This can also serve as a base for krang

**Comment by rabbitsmith** [comment]
This can be very useful. @vktr please add this library as a new project.

**Comment by vktr** [comment]
I agree that this might be a powerful feature, and it will allow people to more easily integrate Hadouken in their own projects.

The work is simple, however I only have a limited amount of time. I'll add it to the feature list and perhaps something will be released later this year, but if anyone feel like taking it upon themselves to lead this, I'll be more than happy to provide support and/or infrastructure.

**Comment by ghost** [comment]
Have a look here
https://github.com/rbarbe/HadoukenApi
https://ci.appveyor.com/project/rbarbe/hadoukenapi/build/artifacts

**Comment by vktr** [comment]
Awesome, @rbarbe :+1:
